packages/commonwealth/client/scripts/views/pages/notification_settings/helper_components.tsx

```diff
@@ -51,9 +53,7 @@ const getTextRows = (
           </CWText>
         </div>
         <CWText type="caption" className="subscription-body-text" noWrap>
-          {renderQuillTextBody(subscription.Thread.body, {
-            hideFormatting: true,
-          })}
```

Comment:
This caused some wacky rendering. From Prod, it looks like we only need the first line, and no adherence to formatting. LMK if i'm wrong about that.